libs/vkd3d/device.c

```diff
@@ -70,8 +70,10 @@ struct vkd3d_optional_extension_info
 
 static const struct vkd3d_optional_extension_info optional_instance_extensions[] =
 {
+#ifdef VKD3D_ENABLE_DEBUG_UTILS
```

Comment:
Empty array will fail MSVC.

Reply:
Thanks for the heads up. I'll work on getting proper CI for MSVC soon so this can be avoided in the future.